packages/position/src/react/__stories__/index.story.js

```diff
@@ -232,3 +232,98 @@ function JsStory({ positionType, ...rest }) {
 JsStory.propTypes = {
   positionType: PropTypes.oneOf(Object.keys(positionFns)).isRequired
 }
+
+const edgeCaseSotries = storiesOf('Position Edge', module)
+
+Object.values(positionComponents).forEach(Comp => {
+  const { displayName } = Comp
+  function Demo() {
+    const [b1hover, setb1hover] = React.useState(false)
+    const [b2hover, setb2hover] = React.useState(false)
+    const [b3hover, setb3hover] = React.useState(false)
+    const [b4hover, setb4hover] = React.useState(false)
+
+    return (
+      <div
+        style={{
+          display: 'grid',
+          justifyItems: 'start',
+          grid: '1fr 1fr/ 1fr 1fr',
```

Comment:
what does the `/` do in this style?

Reply:
It's required for the css prop grid. Grid is a shorthand rule and the / signifies the division between rows and columns when writing the rules.